.env.template

```diff
@@ -0,0 +1,3 @@
+# get the complete env from other volunteers, please
+BIGQUERY_PROJECT=pycontw-225217
+DATATEAM_DISCORD_WEBHOOK=<>
```

Comment:
if you want to implement the discord-related feature, discord me to get the real token , please.

Reply:
please  "contact me"? :)

Reply:
nitpick: "no new line at end of file"

Reply:
will add the new line~